src/components/JumpIn/JumpIn.tsx

```diff
@@ -35,19 +35,30 @@ const JumpIn = React.memo((props: JumpInProps) => {
       e.preventDefault()
 
       const hasLauncher = await launchDesktopApp(desktopAppOptions)
-      !hasLauncher && setIsModalOpen(true)
 
       onTrack?.({
+        type: hasLauncher ? "JUMP_IN" : "OPEN_DOWNLOAD_MODAL",
```

Comment:
What do you thing if we add the strings into a enum in `JumpIn.types.ts`
something like 
```
enum JumpInEventType {
  JumpIn = "jump in",
  OpenDownloadModal = "open download modal",
  Download = "download",
}
```

and then 
```suggestion
type: hasLauncher ? JumpInEventType.JumpIn : .JumpInEventType.OpenDownloadModal,
```